lib/cocoapods-core/specification/root_attribute_accessors.rb

```diff
@@ -146,6 +146,20 @@ def prepare_command
           command.strip_heredoc.chomp if command
         end
 
+        # @return [Bool] Indicates, that if use_frameworks! is specified, the
+        #         framework should include a static library.
+        #
+        def static_framework?
+          attributes_hash['static_framework']
+        end
+
+        # @return [Bool] Indicates, that if use_frameworks! is specified, the
+        #         framework should include a static library.
+        #
+        def static_framework
```

Comment:
It comes from:

```ruby
        def value_for_attribute(attribute)
          if attribute.root_only?
            consumer.spec.send(attribute.name)
          else
            consumer.send(attribute.name) if consumer.respond_to?(attribute.name)
          end
        rescue => e
          results.add_error('attributes', "Unable to validate `#{attribute.name}` (#{e}).")
          nil
        end
```

It will automatically try to invoke a method with the name of the attribute to receive the value.